api/version.go

```diff
@@ -10,4 +10,4 @@
 package api
 
 // VERSION identifies podium's current version
-var VERSION = "5.5.2"
+var VERSION = "5.6.0"
```

Comment:
We'll use major bump because we added some structural changes (updated echo, using extensions...) use 6.0.0 instead